src/test/java/com/exasol/adapter/sql/UserInformationTest.java

```diff
@@ -17,20 +17,57 @@
 
 @ExtendWith(MockitoExtension.class)
 class UserInformationTest {
-    @Mock
-    private final ColumnMetadataReader columnMetadataReaderMock = mock(ColumnMetadataReader.class);
+    private static final String DEFAULT_MASK_WITH_PUBLIC_VALUE = "9223372036854775808";
```

Comment:
Hexadecimal representation is probably easier to understand here.

Reply:
It's a string which will be inserted to the query. Should we use hexadecimal in our queries?

Reply:
I just realized that our SQL does not seem to support hexadecimal literals directly -- needs `TO_NUMBER()` instead.
So for performance reasons the answer is "no".